plugins/table/src/Table.css

```diff
@@ -25,6 +25,16 @@
 .superset-legacy-chart-table .dt-metric {
   text-align: right;
 }
+.superset-legacy-chart-table td.dt-is-filter {
+  cursor: pointer;
+}
+.superset-legacy-chart-table td.dt-is-filter:hover {
+  background-color: linen;
```

Comment:
curious how these were chosen? should we get any design feedback?

Reply:
The [original color](https://github.com/apache/incubator-superset/issues/8273) is:

```
  background-color: lighten(desaturate(@brand-primary, 50%), 50%);
```
https://github.com/apache/incubator-superset/pull/9593/commits/cbf38ff466e81f33efd828d80bdf291391437651#diff-9479060a8cb28b444d0ce7f5df0a33ccL577


![image](https://user-images.githubusercontent.com/335541/79802654-34b43a80-8315-11ea-8e41-416119f5441c.png)

Which translates to `#bce9e5`. I just picked a named color that is close to it and does not look bad. Wanted to use the original color, but it kind of looked too dark to me.

This will probably eventually become customizable with the introduction of the [superset-ui/style](https://github.com/apache-superset/superset-ui/tree/master/packages/superset-ui-style
) package.